### ISSUE TITLE
Section with Inputs does not render as expected on IOS in the dark mode

### ISSUE THREAD

**Issue by RickCastle2018** [Telegram-Mini-Apps/TelegramUI#83]
Hi,

Right here in the documentation (https://tgui.xelene.me/?path=/docs/blocks-section--documentation&globals=platform:ios;theme:dark) we see a neat example usage of Section with Input fields inside. 

<img width="514" alt="image" src="https://github.com/user-attachments/assets/8d75026a-5d14-4150-9f87-95a3e9c35737">

However, for some reason, in my app for every other state (android+light,ios+light,android+dark) it renders just as in the playground, except for when it's IOS and dark mode:

![telegram-cloud-photo-size-2-5300926582245942188-y](https://github.com/user-attachments/assets/16d6507d-080b-4235-80ae-654ebcd607d2)

I wonder, if you may know what might have caused it, where could I fix it, or if it's a bug we could fix and PR. Also, to mention additionally, in Chrome, while in local development all the styles render as intended, it only breaks inside Telegram.

Thanks

**Comment by mainsmirnov** [comment]
Hey, I assume the problem is inside of the SDK you use for bridge with Telegram UI. We use nested color palette and if library does not set it correctly then you would have some issues.

Try https://github.com/Telegram-Mini-Apps/TGUI-Example or t.me/tgui_example_bot/app

**Comment by RickCastle2018** [comment]
It was a long and an extremely painful journey. I didn't start with `@telegram-apps/create-mini-app@latest`, I've been using Preact for this project. So, I just used `@telegram-apps/sdk` and your UI lib, not `react-sdk`. 

I've inited the create-mini-app thing. And saw some literal magic, which is undocumented (or I'm just stupid)... So, I copied your initialization function from there and it started to work after a couple other fixes in my app. I don't know what really happened there... but it seems so without create-mini-app and in particular the react-sdk this UI lib won't be working as intended - as it was purely used in your example here - https://github.com/Telegram-Mini-Apps/TGUI-Example.

Thanks.
